### PR TITLE
Changed domain from `discordapp.com` to `discord.com`

### PR DESCRIPTION
**Pull request by C0braD3v**
- Changed domain from `discordapp.com` to `discord.com` in all locations besides CDN links, to reflect the new Discord changes that will be mandatory on November 7th
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+[![GoDoc](https://godoc.org/github.com/bwmarrin/discordgo?status.svg)](https://godoc.org/github.com/bwmarrin/discordgo) [![Go report](http://goreportcard.com/badge/bwmarrin/discordgo)](http://goreportcard.com/report/bwmarrin/discordgo) [![Build Status](https://travis-ci.org/bwmarrin/discordgo.svg?branch=master)](https://travis-ci.org/bwmarrin/discordgo) [![Discord Gophers](https://img.shields.io/badge/Discord%20Gophers-%23discordgo-blue.svg)](https://discord.gg/0f1SbxBZjYoCtNPP) [![Discord API](https://img.shields.io/badge/Discord%20API-%23go_discordgo-blue.svg)](https://disocrd.com/invite/discord-api)
```
Comment (ewohltman) [inline README.md:3]: Typo in URL here (disocrd instead of discord).

Comment (C0braD3v) [reply README.md]: Resolved per [682664e](https://github.com/bwmarrin/discordgo/pull/769/commits/682664ef8f061f5fff160807ba4b21c6daea02fd)

**Comment by C0braD3v** [comment]
Re-based to `develop` branch :+1:

**Comment by Strum355** [comment]
This commit broke develop branch. 
```
..\github.com\bwmarrin\discordgo\wsapi.go:847:19: (*Session).Close redeclared in this block
        previous declaration at ..\github.com\bwmarrin\discordgo\wsapi.go:841:6
```
Gonna open a PR to fix

**Pull request merged**